src/SFA.DAS.ApprenticeCommitments.Api/Extensions/ApplicationBuilderExtensions.cs

```diff
@@ -36,5 +39,16 @@ static async Task Handler(HttpContext context)
             });
             return app;
         }
+
+        private static string CreateErrorResponse(IEnumerable<ValidationFailure> errors)
+        {
+            var errorList = new Dictionary<string, string>();
+            foreach (var error in errors)
+            {
+                errorList.Add(error.PropertyName, error.ErrorMessage);
+            }
```

Comment:
`IEnumerable` has a `ToDictionary` extension that may clarify the intention here?

e.g.
`    var errorList = errors.ToDictionary(x => x.PropertyName, x => x.ErrorMessage);`

Reply:
good spot, I'll do that